go/exocom/src/security/security_test.go

```diff
@@ -2,13 +2,44 @@ package security_test
 
 import (
 	"github.com/Originate/exocom/go/exocom/src/security"
+	"github.com/Originate/exocom/go/structs"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
 var _ = Describe("Security", func() {
-	It("creates a security manager", func() {
-		securityManager := security.NewSecurityManager()
-		Expect(securityManager).ToNot(BeNil())
+	Describe("ReceiveMessage", func() {
+		var (
+			manager         *security.Manager
+			incomingMessage structs.Message
+			securityRequest *security.Result
+			securityResult  *security.Result
```

Comment:
I don't think this var is needed since securityResult is not shared across code blocks